RecoTauTag/RecoTau/interface/RecoTauCommonUtilities.h

```diff
@@ -88,6 +101,26 @@ RefVectorType castView(const edm::Handle<BaseView>& view) {
   return output;
 }
 
+
+/// Convert a BaseView (View<T>) to a TRefToBaseVector
+template<typename RefVectorType, typename BaseView>
+RefVectorType castViewToOtherBase(const edm::Handle<BaseView>& view) {
```

Comment:
I didn't know that such things are possible :-)...

Reply:
In fact, thanks to you commenting on that, I realise that we don't even need that function anymore since I could simplify the code in RecoTauProducer... so I'll remove it (and the commented code in RecoTauProducer)